backend/swagger/item.go

```diff
@@ -0,0 +1,89 @@
+package swagger
+
+import (
+	"context"
+	"github.com/SAKA-club/todo/backend/errs"
+	"github.com/SAKA-club/todo/backend/gen/models"
+	"github.com/SAKA-club/todo/backend/gen/restapi/operations"
+	"github.com/SAKA-club/todo/backend/gen/restapi/operations/item"
+	"github.com/go-openapi/runtime/middleware"
+	"time"
+)
+
+type ItemService interface {
+	GetAll(ctx context.Context) ([]*models.Item, error)
+	Get(ctx context.Context, ID int64) (*models.Item, error)
+	Create(ctx context.Context, title string, body string, priority bool, scheduleTime time.Time, completeTime time.Time) (*models.Item, error)
+	Delete(ctx context.Context, ID int64) error
+	Update(ctx context.Context, ID int64, title string, body string, priority bool, scheduleTime time.Time, completeTime time.Time) (*models.Item, error)
+}
+
+func Item(api *operations.TodoAPI, service ItemService) {
+	api.ItemGetAllHandler = item.GetAllHandlerFunc(func(params item.GetAllParams) middleware.Responder {
+		ctx := params.HTTPRequest.Context()
+		//log.Ctx(ctx).With()
+		result, err := service.GetAll(ctx)
+		if err != nil {
+			return item.NewGetNotFound().WithPayload(err)
```

Comment:
This doesn't look like it returns the expected errorResponse defined in your swagger.

Reply:
Something to the effect of `MyResponse{Code: 400, Message: "Here is a message"}`

Reply:
todo:
swagger created a model and a struct that has 3 properties 
create an instance of a struct and populate each one of the properties